packages/core/core-flows/src/order/workflows/exchange/confirm-exchange-request.ts

```diff
@@ -58,6 +61,29 @@ export const confirmExchangeRequestValidationStep = createStep(
   }
 )
 
+/**
+ * This step confirms that a requested exchange has atleast one item to return or send
+ */
+const confirmIfExchangeItemsArePresent = createStep(
+  "confirm-if-exchange-items-are-present",
+  async function ({
+    exchangeItems,
+    returnItems,
+  }: {
+    exchangeItems: OrderExchangeItemDTO[]
+    returnItems?: OrderReturnItemDTO[]
+  }) {
+    if (exchangeItems.length > 0 || (returnItems || []).length > 0) {
```

Comment:
q: should an exchange have both inbound and outbound items?

Reply:
I would say yes 👍